client/extensions/woocommerce/app/products/product-form-variation-card.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import ProductVariationTypesForm from './product-variation-types-form';
+import FormToggle from 'components/forms/form-toggle';
+
+export default class ProductFormVariationCard extends Component {
+
+	static propTypes = {
+		product: PropTypes.shape( {
+			id: PropTypes.isRequired,
+			type: PropTypes.string.isRequired,
+			name: PropTypes.string,
+		} ),
+		editProduct: PropTypes.func.isRequired,
+		editProductAttribute: PropTypes.func.isRequired,
+	};
+
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			isVariation: props.product && 'variable' === props.product.type ? true : false,
+		};
+
+		this.handleToggle = this.handleToggle.bind( this );
+	}
+
+	handleToggle() {
```

Comment:
See also https://github.com/Automattic/wp-calypso/pull/13108#pullrequestreview-33639805 regarding having a helper for such things